src/applications/ivc-champva/10-10d-extended/chapters/applicantInformation.js

```diff
@@ -772,7 +773,8 @@ export const applicantPages = arrayBuilderPages(
     }),
     page18: pageBuilder.itemPage({
       path: 'applicant-relationship/:index',
-      title: item => `${applicantWording(item)} relationship to the sponsor`,
+      title: item =>
+        `What's ${applicantWording(item)} relationship to the Veteran?`,
```

Comment:
Can we normalize the apostrophe here? It would be better to utilize `’` instead of `'`.